byu_awslogin/util/data_cache.py

```diff
@@ -1,10 +1,36 @@
+import codecs
 import configparser
+import datetime
 import os
+import pickle
 from os.path import expanduser
-import datetime
+
 from .consoleeffects import Colors
 
 
+def load_cached_adfs_auth():
+    file = _aws_file('config')
+    config = _open_config_file(file)
+    section = 'all'
+    if config.has_section(section) and config.has_option(section, 'adfs_auth'):
+        unpickled = pickle.loads(codecs.decode(config[section]['adfs_auth'].encode(), "base64"))
+        return unpickled
+    else:
+        return None
+
+
+def cache_adfs_auth(adfs_auth_result):
```

Comment:
how sensitive is this information? would it be better to store in the .aws/credentials file or does it not matter?

Reply:
It contains the cookies set by the ADFS login, so it definitely is sensitive information in that it constitutes an active session. Both files are in plaintext on the user's computer, so it doesn't really add much security to put it in the credentials file. However, I do think it probably does fit in better there since it is authentication information, so I'll move it over there.